src/integrationTest/java/uk/gov/hmcts/darts/task/service/OutboundAudioDeleterProcessorTest.java

```diff
@@ -404,6 +406,70 @@ void whereLastAccessedIsNullUseCreatedAtAndInProgressStatus() {
         assertEquals(EXPIRED, tod.getTransformedMedia().getMediaRequest().getStatus());
     }
 
+    @Test
+    void whereLastAccessedWasOneDayAgo() {
+        whereLastAccessedWasOneDayAgoSupport(false, false);
+    }
+
+    @Test
+    void whereLastAccessedWasOneDayAgoButMediaIsHidden() {
+        whereLastAccessedWasOneDayAgoSupport(true, false);
+    }
+
+    @Test
+    void whereLastAccessedWasOneDayAgoButMediaIsDeleted() {
+        whereLastAccessedWasOneDayAgoSupport(false, true);
+    }
+
+    @Test
+    void whereLastAccessedWasOneDayAgoButMediaIsHiddenAndDeleted() {
+        whereLastAccessedWasOneDayAgoSupport(true, true);
+    }
+
+    void whereLastAccessedWasOneDayAgoSupport(boolean isHidden, boolean isDeleted) {
+        HearingEntity hearing = dartsDatabase.createHearing(
+            "NEWCASTLE",
+            "Int Test Courtroom 2",
+            "2",
+            HEARING_DATE
+        );
+
+
+        // Last accessed on a 2023-10-20 friday
+        MediaRequestEntity currentMediaRequest = MediaRequestTestData.createCurrentMediaRequest(
```

Comment:
I noticed this method is deprecated. Could you refactor the new usages?

Reply:
Updated :)